plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -38,6 +41,19 @@ export async function createBackendRouter(
   const { eventBroker, config, logger, discovery, catalogApi, urlReader } =
     args;
 
+  // Avoid hard coded path
+  // https://issues.redhat.com/browse/FLPATH-932
+  const openapiFolderPath = path.join(
```

Comment:
On second thought, I think this relative path won't work in production and might even break the plugin during the initialization. Let me think about it.

Reply:
Correct, it will not work. I was dealing with a similar issue in the Notifications plugin and the safest and simplest solution I could find was in passing the document as a JavaScript object right away.

With this approach, we embed the API specification at the plugin build time before publishing it, so we have full control without any assumptions on the target building setup.

In a nutshell: 
- https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications-backend/package.json#L24
- https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications-backend/scripts/openapi.sh#L10
- https://github.com/janus-idp/backstage-plugins/blob/main/plugins/notifications-backend/src/service/router.ts#L51

I am not sure how this approach will work for more complex documents referencing others but I believe there must be a way how to do it with a similar approach.

Reply:
Thank you for finding this issue @mareklibra and @caponetto :) 
I addressed the issue as suggest. Can you please take a look?

@caponetto I also added the v2 implementation endpoint to be sure that everything is working fine (see commit https://github.com/janus-idp/backstage-plugins/pull/1123/commits/222ff9dc135bc11e84b85aaa9f9d32f9b2dd47a8 )